lib/Driver/ToolChains.cpp

```diff
@@ -1286,6 +1300,24 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   if (!Linker.empty()) {
     Arguments.push_back(context.Args.MakeArgString("-fuse-ld=" + Linker));
   }
+  
+  // Configure the toolchain
+  // By default, use the system clang++ to link.
+  const char * Clang = "clang++";
```

Comment:
Is there a reason to assume clang++ for the driver?  exherbo for-example won't necessarily have clang++ but rather target-clang++.  Why not use the standard names for the driver `cc`?